libraries/AP_HAL_ChibiOS/hwdef/common/malloc.c

```diff
@@ -371,6 +376,19 @@ void *malloc_axi_sram(size_t size)
 #endif
 }
 
+/*
+  allocate memory for ethernet DMA
+*/
+void *malloc_eth_safe(size_t size)
+{
+#if defined(STM32H7)
+    return malloc_flags(size, MEM_REGION_FLAG_ETH_SAFE);
+#else
+    (void)size;
+    return NULL;
```

Comment:
This stuff testable in SITL?

Do we have any autotests coming for this?

Reply:
SITL support will be in a follow-up PR